packages/rancher-logging/rancher-logging.patch

```diff
@@ -142,13 +142,15 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/values
 +    repository: rancher/banzaicloud-fluentd
 +    tag: v1.11.5-alpine-1
 +
-+fluentbit_tolerations:
-+  - key: node-role.kubernetes.io/controlplane
-+    value: "true"
-+    effect: NoSchedule
-+  - key: node-role.kubernetes.io/etcd
-+    value: "true"
-+    effect: NoExecute
++fluentbit:
++  mountPath: ""
++  tolerations:
++    - key: node-role.kubernetes.io/controlplane
++      value: "true"
++      effect: NoSchedule
++    - key: node-role.kubernetes.io/etcd
++      value: "true"
++      effect: NoExecute
```

Comment:
Want to double check that you ran a `helm lint` here

Reply:
Will do